### PR TITLE
ui: integrated add query flow

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes:

Fixes [#10916](https://github.com/open-metadata/OpenMetadata/issues/10916)

I worked on Integrating new add query flow, and feedback for query feature

image

image

image

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/TableQueries/QueryCard.tsx [hunk 1]
```diff
@@ -128,23 +156,44 @@ const QueryCard: FC<QueryCardProp> = ({
           title={
             <Space className="font-normal p-y-xs" size={8}>
               <Text>{queryDate}</Text>
-              <Text>{SINGLE_DOT}</Text>
-              <Text>{`${t('label.by-lowercase')} ${query.updatedBy}`}</Text>
+              <Text className="text-gray-400">{PIPE_SYMBOL}</Text>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TableQueries/QueryCard.tsx:159]: Did we use same Pipe symbol as EntityInfo

File: openmetadata-ui/src/main/resources/ui/src/components/TableQueries/table-queries.style.less [hunk 2]
```diff
@@ -40,3 +38,23 @@
 .query-card-container.selected {
   border: 1px solid @info-color;
 }
+
+.query-right-panel {
+  position: sticky;
+  height: calc(100vh - @summary-panel-offset);
+  z-index: 9;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TableQueries/table-queries.style.less:45]: We should not modify zIndex, Is it required?

**Comment by ShaileshParmar11** [comment]
> @ShaileshParmar11 when users adding query against a table we should add that table to queryUsedIn by default.

currently, we are showing the selected option in the dropdown, do we not show that and add the table directly in the query used in and if users select more tables we also add that?

File: openmetadata-ui/src/main/resources/ui/src/components/TableQueries/QueryCard.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/TableQueries/QueryCard.tsx:159]: the symbol is the same, but in entity info, we have added it from localization. and we are avoiding that, right? to add symbol in localized file so I have added const for symbol

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/TableQueries/QueryCard.tsx:159]: Let's make a note and remove it in next release

File: openmetadata-ui/src/main/resources/ui/src/components/TableQueries/table-queries.style.less (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/TableQueries/table-queries.style.less]: Thanks for pointing this out, and yes we don't need z-index here!

**Review by chirag-madlani — APPROVED** [review]

**Review by harshach — APPROVED** [review]

**Pull request merged**